api-proxy/src/logging.mjs

```diff
@@ -0,0 +1,28 @@
+import * as nu from 'util'
+
+const create = (logStream = process.stderr) => {
+  let lastStamp = -Infinity
+
+  const truncate = ms => Math.floor(ms / 60e3) * 60e3
```

Comment:
Codacy says this style of number can be inaccurate. However, (1) I disagree, (2) It wouldn't matter if they were, and (3) I think it is easier to see the value when not crowded with zeros.